Xamarin.Forms.Core/StateTriggerBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel;
+
+namespace Xamarin.Forms
+{
+	public class StateTriggerBase : BindableObject
```

Comment:
abstract?

Reply:
Oddly enough the class this design was taken from isn't abstract (albeit I agree it makes sense, apart from that would diverge on a common design): https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.StateTriggerBase